inc/compat/themes/compat-theme-Avada.php

```diff
@@ -115,6 +116,19 @@ public function change_sticky_elements_relative_header( $attributes ) {
 	 * @param  array  $css_variables  The CSS variables key/value pairs.
 	 */
 	public function add_css_variables( $css_variables ) {
+		if ( function_exists( 'awb_get_fusion_settings' ) ) {
+			// Get theme settings
+			$theme_settings = awb_get_fusion_settings();
+
+			// Retrieve danger accent color from the settings
+			$danger_accent_color = $theme_settings->get( 'danger_accent_color' );
+		}
+
+		// Define default color if not set
+		if ( empty( $danger_accent_color ) ) {
+			$danger_accent_color = '#12b878';
```

Comment:
@vlad-dedikov, to simplify this code you could always set the variable with this default value at the top of this function, then change the value from the theme settings if the theme settings function is available.

Also, is this default color value set by the theme?

Reply:
@diegoversiani Moved to the Avada Builder compat file.